src/include/miopen/execution_context.hpp

```diff
@@ -100,12 +101,11 @@ struct ExecutionContext
         // clang-format off
         return GetUserDbPath()
 #if MIOPEN_ENABLE_SQLITE
-             + "/miopen.udb";
+             + "miopen_" + std::string{MIOPEN_PERFDB_SCHEMA_VER} + ".udb";
```

Comment:
[tip] You can simply use `string{"1.0.0"}` right here.

Reply:
I don't think burying version strings in the code is a good idea.